services/core-web/common/actionCreators/projectSummaryActionCreator.js

```diff
@@ -0,0 +1,163 @@
+import { notification } from "antd";
+import { showLoading, hideLoading } from "react-redux-loading-bar";
+import { request, success, error } from "../actions/genericActions";
+import * as reducerTypes from "../constants/reducerTypes";
+import * as Strings from "../constants/strings";
+import * as projectSummaryActions from "../actions/projectSummaryActions";
+import * as API from "../constants/API";
+import { ENVIRONMENT } from "../constants/environment";
+import { createRequestHeader } from "../utils/RequestHeaders";
+import CustomAxios from "../customAxios";
+
+export const createProjectSummary = ({ mineGuid }, payload) => (dispatch) => {
+  const message = "Successfully created a new project summary";
+  dispatch(request(reducerTypes.CREATE_MINE_PROJECT_SUMMARY));
+  dispatch(showLoading("modal"));
+  return CustomAxios()
```

Comment:
can you update all 'showLoading' and 'hideLoading' within this file to remove the "modal" scope? Since we aren't using a modal, this isn't needed.

ie 'dispatch(showLoading("modal"));' -> 'dispatch(showLoading());'